src/demo_gen/custom_functions.md

```diff
@@ -0,0 +1,45 @@
+# Making Custom Functions
+
+Let's say you want to add your own custom demonstration functions for a given struct, without demo_gen automagically creating the content of JS functions for you.

```

Comment:
I still don't really understand what a "custom demonstration function" would mean here. What kind of thing can one achieve with this?